$OpenBSD: patch-src_map_krovak_h,v 1.2 2020/01/24 08:02:36 landry Exp $
https://github.com/tumic0/GPXSee/issues/266

../src/map/krovak.h:21:17: error: expected member name or ';' after declaration specifiers
        double _e, _A, _B, _t0, _n, _r0, _phiP;
        ~~~~~~         ^
/usr/include/ctype.h:52:12: note: expanded from macro '_B'
#define _B      0x80

Index: src/map/krovak.h
--- src/map/krovak.h.orig
+++ src/map/krovak.h
@@ -18,7 +18,7 @@ class Krovak : public CT (public)
 	virtual Coordinates xy2ll(const PointD &p) const;
 
 private:
-	double _e, _A, _B, _t0, _n, _r0, _phiP;
+	double _e, _A, B, _t0, _n, _r0, _phiP;
 	double _cosAlphaC, _sinAlphaC, _lambda0, _FE, _FN;
 };
 
